src/lib/holocene/toaster.svelte

```diff
@@ -16,9 +17,15 @@
   class="fixed bottom-5 right-5 z-[99999] flex flex-col items-end gap-2"
   role="log"
 >
-  {#each $toasts as { message, variant, id } (id)}
+  {#each $toasts as { message, variant, id, link } (id)}
     <ToastComponent {closeButtonLabel} {variant} {id} on:dismiss={dismissToast}>
-      {message}
+      {#if link}
```

Comment:
I could be convinced that should be it's own component or something someday. But, today doesn't have to be that day.

Reply:
Ehhh this is so simple that i don't think the juice is worth the squeeze, but if it gets any more complicated for sure.